### PR TITLE
Employee Offboarding Pack

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24690

## Description
Created a pack for Employee Offboarding.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
5.0.0

## Does it break backward compatibility?
No

### CODE REVIEW

**Comment by idovandijk** [comment]
@yaron-libman we just need https://github.com/demisto/content/blob/eb2abd84203a3b8fb4ef234f2d3bfc577b521237/Packs/EmployeeOffboarding/ReleaseNotes/1_0_0.md

and description in https://github.com/demisto/content/blob/eb2abd84203a3b8fb4ef234f2d3bfc577b521237/Packs/EmployeeOffboarding/pack_metadata.json 

thanks

**Comment by yaron-libman** [comment]
Reviewed and approved.

File: Packs/EmployeeOffboarding/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "created": "2020-05-19T13:07:29Z",
+    "categories": [
+        "IT Services"
+    ],
+    "tags": [],
+    "useCases": [],
```
Comment (michalgold) [inline Packs/EmployeeOffboarding/pack_metadata.json:14]: use case should be: Employee Offboarding

**Comment by idovandijk** [comment]
@anara123 we can merge

**Pull request merged**